src/scss/components/_donors.scss

```diff
@@ -0,0 +1,61 @@
+.donors_info {
+  @include grid;
+}
+
+.donors {
+  position: relative;
+  overflow-x: hidden;
+}
+
+.donors_image {
+  width: 100%;
+}
+
+.donors_description {
+  @include body-xl-med;
+
+  font-size: 2.5rem;
+  @include col(2, 8);
+}
+
+.donors_link_container {
+  @include link-typography;
+  @include col(2, 5);
+
+  display: flex;
+  padding: 2.5rem 0 19.5rem;
+}
+
+.donors_ctaPrimary_link a {
+  color: #fff;
+  border-radius: 100px;
+  padding: 1.25rem 5.4rem;
+  background-color: #08152e;
+  margin-right: 3rem;
+}
+
+.donors_ctaSecondary_link a {
+  position: relative;
+  color: #08152e;
```

Comment:
You can find this color as a variable in the _variables.scss file, here you can set it up as background-color: var(--brand-navy)

Reply:
thanks, i corrected it!!